widgets/common/src/DynamicDropdown.tsx

```diff
@@ -58,6 +58,7 @@ export default function DynamicDropdown({
     className = '',
     disabled = false,
     fetchAll = false,
+    searchParam = ['_search'],
```

Comment:
I would call this property

```suggestion
    searchParams = ['_search'],
```

since it's an array and can contain multiple params :slightly_smiling_face: